services/QuillLMS/app/models/concept_feedback.rb

```diff
@@ -32,17 +32,20 @@ class ConceptFeedback < ApplicationRecord
   has_many :translation_mappings, as: :source
   has_many :english_texts, through: :translation_mappings
   has_many :translated_texts, through: :english_texts
+  has_many :gengo_jobs, through: :english_texts
   store_accessor :data, :description
 
   after_commit :clear_concept_feedbacks_cache
 
   def cache_key = "#{ALL_CONCEPT_FEEDBACKS_KEY}_#{activity_type}"
 
   def as_json(options=nil)
-    translation = translation(locale: "es-la")
+    source_api = options&.dig(:source_api) || TranslatedText::OPEN_AI_SOURCE
+    locale = Gengo::SPANISH_LOCALE
```

Comment:
I think that `locale = Gengo::SPANISH_LOCALE` can be removed and put as a default parameter in the `translate!` method.